code_review/16-09-20_s2s_util.py

```diff
@@ -0,0 +1,129 @@
+"""
+Utility functions for seq2science
+"""
+import os
+from typing import List
+
+import pandas as pd
+import pysradb
+from snakemake.io import expand
+
+
+def _sample_to_idxs(df: pd.DataFrame, sample: str) -> List[int]:
+    """
+    Get the index/indices that belong to a run in a pysradb dataframe
```

Comment:
What does `idxs` represent? Is it a kind of ID or just a df row number of the pysradb dataframe?

Reply:
It's a list of the row number(s), which at the same time is the dataframe index.

So we can index the dataframe like this: `df.loc[idxs]`